### PR TITLE
DOCK-2290: updated navbar dropdown

### PR DESCRIPTION
**Pull request by hyunnaye**
**Description**
Updated dropdown on navbar to look like the zeplin mockups.

UI after changes:
Screenshot from 2023-01-19 16-52-26
Screenshot from 2023-01-19 16-52-34

Zeplin mockups
https://app.zeplin.io/project/5fc690a2717ee9144814f190/screen/6022acb6632937281e6935ec. 

**Review Instructions**
Verify that the new changes represent the mockups accurately and the integration tests still pass.

**Issue**
https://github.com/dockstore/dockstore/issues/5261

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone.

### CODE REVIEW

File: src/app/navbar/navbar.component.scss
Comment (coverbeck) [inline src/app/navbar/navbar.component.scss:84]: Aren't we supposed to avoid pixels and use rems instead? It's in the [opinionated style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide), although at one point I thought Gary mentioned it may not be necessary any more. @kathy-t may know?

If rems aren't required, can you please update the style guide, so I don't keep making comments like this. If they are required, please update here.

Comment (hyunnaye) [reply src/app/navbar/navbar.component.scss:84]: I was thinking about that too but wanted to make it the same padding as the notification bell and My Dockstore buttons which had 14px as padding. https://github.com/dockstore/dockstore-ui2/blob/e5d682ce2906cd65b99dc5be0b41861404244f87/src/app/navbar/navbar.component.scss#L21

If we should use rem insread, i'll change it for those as well or consistency!

Comment (kathy-t) [reply src/app/navbar/navbar.component.scss:84]: Hmm, I generally follow the style guide and use rem. I don't really see a downside to using REM because it makes the design more accessible

**Review by svonworl — APPROVED** [review]

**Review by kathy-t — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

File: cypress/e2e/immutableDatabaseTests/dropdown.ts
Comment (coverbeck) [inline cypress/e2e/immutableDatabaseTests/dropdown.ts:435]: This isn't going to work with my upcoming Cypress 12 PR :( . You had no way of knowing that, and we have this pattern all over -- I've been changing them.

I can fix it after you merge, or if you want to get it ready in advance...

In Cypress 12, different tests are isolated by default. This test currently assumes you are

1. On a page, carried over from the previous test
2. Logged in

In Cypress 12
1. You won't be on a page because you need to explicitly navigate to one
2. You won't be logged in (local storage, with the token, is wiped between tests).

There are two possible fixes:

1. Get rid of lines 433 and 434 (and their closing braces), so that the previous page and logged in token "carry over".
2. Keep lines 433 and 434, but do a `cy.visit` and the call to set the user token.

I will probably just do 1 because it's easier; if you feel strongly and want to keep the log out as a separately named test, then could you please do 2.

**Review by coverbeck — APPROVED** [review]
Approved, but with an option if you don't want me to change your code shortly after it's merged. :)

File: cypress/e2e/immutableDatabaseTests/dropdown.ts
Comment (hyunnaye) [reply cypress/e2e/immutableDatabaseTests/dropdown.ts]: I'll change it!

**Pull request merged**
